src/js/dropdown-menu/specs/dropdown-menu-view-spec.js

```diff
@@ -102,7 +102,9 @@ define([
                         text: 'username',
                         url: 'dashboard'
                     },
-                    button_label: 'User options dropdown',
+                    button: {
+                        label: 'User options dropdown'
```

Comment:
It would be good to add some tests for a dropdown with an icon. In particular, test that clicking on the icon has the same behavior as clicking on the button.